lld/ELF/InputFiles.cpp

```diff
@@ -1712,8 +1728,8 @@ BitcodeFile::BitcodeFile(MemoryBufferRef mb, StringRef archiveName,
   // symbols later in the link stage). So we append file offset to make
   // filename unique.
   StringRef name = archiveName.empty()
-                       ? saver().save(path)
-                       : saver().save(archiveName + "(" + path::filename(path) +
+                       ? saved_symbol(path)
+                       : saved_symbol(archiveName + "(" + path::filename(path) +
```

Comment:
Why not just use unique_saver().save() directly like you do below on line 1764?

Reply:
I was thinking of `BitcodeFile::saved_symbol` as helper functions to make it more explicit that bitcode file uses unique saver. 

Now I use `saver()` or `unique_saver()` from linker context directly (for simpler code) and add comment around to explain the motivation.